tracing/opentracing/options.go

```diff
@@ -6,7 +6,9 @@ package grpc_opentracing
 import (
 	"context"
 
-	"github.com/opentracing/opentracing-go"
+	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
isn't this redundant?

Reply:
This is a goimports thing. If I actually want to propose this for review I should clean this up.